index.js

```diff
@@ -1,61 +1,78 @@
+  next()
+})
 
+// Use Redis for session persistence.
+// Remove the following 2 lines and the 'store' property if you prefer an in-memory store.
+const RedisStore = require('connect-redis')(session)
+const redisServer = process.env.REDIS_SERVER_ADDRESS || 'localhost'
+const redisClient = redis.createClient({ host: redisServer })
+redisClient.on('error', console.error)
 app.use(session({
-  secret:            config.server.sessionSecret,
-  resave:            false,
+  store: new RedisStore({ client: redisClient, prefix: `session-${config.organisation.agb}:` }),
+  secret: config.server.sessionSecret,
+  resave: false,
+  name: `session-${config.organisation.agb}`,
   saveUninitialized: false
-}));
+}))
 
-app.use(express.static('public'));
-app.use(express.json());
-app.use(bodyParser.json());
+app.use(express.static('public'))
+app.use(express.json())
+app.use(bodyParser.json())
 
-app.use('/api',      clientAPI);
-app.use('/external', externalAPI);
+app.use('/api', clientAPI)
+app.use('/external', externalAPI)
+app.use('/sso', ssoHandler)
 
-eventAPI(io);  // Mount events API using socket.io
+eventAPI(io) // Mount events API using socket.io
 
 server.listen(config.server.port, () =>
-  Logger.log(`Server is listening on port ${config.server.port}`));
+  Logger.log(`Server is listening on port ${config.server.port}`))
 
 // Register our organisation with the Nuts node on startup
 
-Logger.log(`Registering our organisation ${config.organisation.name}`);
+Logger.log(`Registering our organisation ${config.organisation.name}`)
 
 crypto.getPublicKey(config.organisation.agb)
-.then(() => Logger.log("Organisation already registered"))
-.catch(e => {
-  if ( !e.response || e.response.status != 404 )
-    return Logger.error("Error registering organisation, is your Nuts node up?",e);
-
-  crypto.generateKeyPair(config.organisation.agb)
-  .then(pubKey => Logger.log(`Registered! Public key for copy'n'pastin:\n${JSON.stringify({
-    name: config.organisation.name,
-    identifier: `urn:oid:2.16.840.1.113883.2.4.6.1:${config.organisation.agb}`,
-    publicKey: pubKey.replace(/\n/g, "\n")
-  })}`))
+  .then(() => Logger.log('Organisation already registered'))
   .catch(e => {
-    Logger.error("Error registering organisation, is your Nuts node up?",e);
-  });
-});
+    if (!e.response || e.response.status != 404) { return Logger.error('Error registering organisation, is your Nuts node up?', e) }
+
+    crypto.generateKeyPair(config.organisation.agb)
+      .then(pubKey => {
+        const exampleVendorClaimEvent = {
+          type: 'VendorClaimEvent',
+          payload: {
+            vendorIdentifier: '',
+            orgName: config.organisation.name,
+            orgIdentifier: `urn:oid:2.16.840.1.113883.2.4.6.1:${config.organisation.agb}`,
+            orgKeys: [pubKey]
+          }
+        }
```

Comment:
👍🏻

Zou wel even de naam `exampleVendorClaimEvent` veranderen 😉

Reply:
Het is immers bedoeld als voorbeeld. Als in: hoe kun je je organisatie toevoegen aan het register, hier is een voorbeeld van de vendorClaim.